server/policy_test.go

```diff
@@ -151,3 +166,113 @@ func TestEvaluatePolicyFailure(t *testing.T) {
 		})
 	}
 }
+
+func TestSevSnpSignedUefi(t *testing.T) {
+	// Generate fake signed reference measurements.
+	fw := ovmfsev.CleanExample(t, 0x200000)
+	meas, err := gcesev.LaunchDigest(gcesev.LaunchOptionsDefault(), fw)
+	if err != nil {
+		t.Fatal(err)
+	}
+	certdir := t.TempDir()
+	keydir := t.TempDir()
+	outdir := t.TempDir()
+	outdir2 := t.TempDir()
```

Comment:
nit: can you define these closer to where they are used?

Reply:
Removed outdir2.